### ISSUE TITLE
Error occurred in starting fork on building GW locally

### ISSUE THREAD

**Issue by nbdnnm** [GraphWalker/graphwalker-project#111]
Given:
The latest GW snapshot is building on windows 10, maven version info is
Apache Maven 3.3.9 (bb52d8502b132ec0a5a3f4c09453c07478323dc5; 2015-11-10T17:41:47+01:00)
Maven home: C:\Program Files (x86)\apache-maven-3.3.9\bin\..
Java version: 1.8.0_73, vendor: Oracle Corporation
Java home: C:\Program Files (x86)\Java\jdk1.8.0_73\jre
Default locale: ru_RU, platform encoding: Cp1251
OS name: "windows 10", version: "10.0", arch: "x86", family: "dos"

When:
Command for building is mvn -e clean package -DskipTest

Then:
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project graphwalker-cli: ExecutionException Error occurred in starting fork, check output in log -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project graphwalker-cli: ExecutionException Error occurred in starting fork, check output in log

Log file is below
[mvn_output.zip](https://github.com/GraphWalker/graphwalker-project/files/628877/mvn_output.zip)

**Comment by nilols** [comment]
Think it should be -DskipTests (or -Dmaven.test.skip) instead of
-DskipTest, will have a look at the zip file as soon as possible

**Comment by nbdnnm** [comment]
Yes, I have found already that it should be -DskipTests, I took it from https://github.com/GraphWalker/graphwalker-project/blob/master/appveyor.yml#L4

And regarding this fork error, I have found the root cause as well.
It is due to this line
https://github.com/GraphWalker/graphwalker-project/blob/master/graphwalker-cli/pom.xml#L82
On my laptop 4 threads so I suppose it fails when tries to create a fifth one.
I see there is possible to set <forkCount>1C</forkCount> which means to take the number of cores

Don't waste your time, I will send a pull request.
